docs/analyzer/text-analytics.md

```diff
@@ -0,0 +1,32 @@
+# Azure Text Analytics Integration
```

Comment:
this file should be located under docs/samples/python/text-analytics

Reply:
Aligned myself with 'developing new recognizers', thought that it might be in the same location as this is an extension of it.
Moving it